packages/fhir-converter/src/templates/cda/Resources/MedicationAdministration.hbs

```diff
@@ -20,12 +20,12 @@
                 "route":{{>DataType/CodeableConcept.hbs code=medicationAdministration.routeCode}}
                 "dose":
                 {
-                    "value":{{medicationAdministration.doseQuantity.value}},
+                    "value":"{{medicationAdministration.doseQuantity.value}}",
```

Comment:
Are we sure to move all these numbers to string? Confirming b/c there's no context for this on the PR description or specific notes

Reply:
yah sorry about that. [Context](https://metriport.slack.com/archives/C04T256DQPQ/p1705998742866599?thread_ts=1705997203.233089&cid=C04T256DQPQ)